modules/gltools/src/context/context.js

```diff
@@ -49,8 +49,10 @@ export function createGLContext(options = {}) {
     if (options.throwOnError) {
       throw new Error(message);
     }
-    return null;
+    // eslint-disable-next-line
+    return console.error(message);
   }
+  options.onError = onError;
```

Comment:
While we can modify the above object rather than mint a new one, it does make this code harder to read. Contexts are rarely created so in my opinion this code should be optimized for readability rather than avoiding small object creation. Naturally, his is just my opinion, disregard if in conflict with the principles of luma.gl.

Reply:
I prefer we get in that habit of avoiding unnecessary allocations, and I don't find this structure hard to read. I know there's generally a policy against modifying arguments in vis.gl though, so a couple of options I see that would satisfy both requirements are:
1. Since the error callback is only used by internal functions, just change the signature of those functions to take `onError` as an explicit argument.
2. Make `onError` a global function in this module that just takes `throwOnError` as an explicit argument. And then just call it directly where it's used.